### PR TITLE
feat: product-detail-page

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added Product Detail Page. Please give your reviews.
Screenshot
image

### CODE REVIEW

**Review by aayushjain-code — COMMENTED** [review]
Everything looks fine navbar needs some padding to make it look more aligned to center.

**Comment by ShekharDhangar** [comment]
> Everything looks fine navbar needs some padding to make it look more aligned to center.

I have made the changes, please check

**Review by MuskanChhatrasal — APPROVED** [review]

File: product-detail-page/product-page.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.product-details-container > *{
+    margin: 1rem 0;
+}
+
+.product-description-box{
+    margin: 1rem 0;
+}
+.product-description{
```
Comment (MuskanChhatrasal) [inline product-detail-page/product-page.css:33]: Try to use relative units. It will be more helpful while adding responsiveness.
Else LGTM!!

**Review by MuskanChhatrasal — APPROVED** [review]

File: product-detail-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+}
+.cta-btn-container .add-wishlist-btn{
+    background-color: transparent;
+    border: 1px solid var(--clr-black);
+    color: var(--clr-black);
+    width: 10%;
+    max-width: 10%;
+}
+
+
```
Comment (Developers-codz) [inline product-detail-page/product-page.css:85]: remove extra spaces

**Review by Developers-codz — APPROVED** [review]
Looking Good to me!

**Review by vrinda-mahajan — APPROVED** [review]
Good to merge.👍

**Pull request merged**